app/budget/directives/modals/budgetConfig/budgetConfig.html

```diff
@@ -0,0 +1,33 @@
+<div class="budget-config">
+	<div class="budget-config__container">
+		<ipa-tabs
```

Comment:
Sometimes it helps to put the first attribute on the opening line, then line the rest up with that attribute. That way it won't line up with the inner content and will be a little easier to read.